src/lib/spinel/spinel.h

```diff
@@ -2755,7 +2755,7 @@ enum
      *
      * Content is
      *   `S` : Key Rotation Time (in units of hour)
-     *   `C` : Security Policy Flags (as specified in Thread 1.1 Section 8.10.1.15)
+     *   `D` : Security Policy Flags (as specified in Thread 1.1 Section 8.10.1.15)
```

Comment:
Suggest to be more specific here in documenting this. `D` really means "blob of data" without `len` being encoded. 

I think would be good to define as this (which keeps it backward compatible):
- `C` Security flag (lower bits)
- `C` Flag extension (higher bits) - This field is optional and may not be present [Then we can describe what is done on `SET` if it is not present and what default values for flags are added].